playwright-e2e/utils/log-utils.ts

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from "@playwright/test";
+
+
+export function logParticpantCreated(participantEmail: string, participantName: string) {
+    test.info().annotations.push({
+        type: "participant",
+        description: participantEmail + ' ' + participantName,
+    });
+}
+
+export function logEmailVerificationResult(emailSubject: string, passed:boolean) {
+    const testResult = passed ? ":heavy_check_mark:" : ":x:";
+    test.info().annotations.push({
+        type: "email with subject '" + emailSubject + "'",
+        description: testResult + " " + emailSubject
+    });
```

Comment:
looks like it'll be handy for debugging tests

Reply:
I had some fun with this--it shows up at the top of the generated playwright html report.